Lib/unittest/mock.py

```diff
@@ -862,7 +862,8 @@ def assert_has_calls(self, calls, any_order=False):
                 not_found.append(kall)
         if not_found:
             raise AssertionError(
-                '%r not all found in call list' % (tuple(not_found),)
+                '%r not all found in call list: %r' % (tuple(not_found),
+                    all_calls)
```

Comment:
I'd also include `self._mock_name` in the error message, like in https://github.com/python/cpython/pull/8205/files#diff-ff75b1b83c21770847ade91fa5bb2525R829

Reply:
What should the message look like? Would it be something along the lines of "%r does not contain all of %r in its call list, found %r instead", where the first format parameter would be self._mock_name or 'mock'?

Reply:
I believe so, yes.  I just committed and edit from the github UI doing that.  awaiting CI test results.